stumpy/stomp.py

```diff
@@ -107,23 +105,22 @@ def _stomp(T_A, m, T_B=None, ignore_trivial=True):
     l = n - m + 1
     excl_zone = int(np.ceil(m / 4))  # See Definition 3 and Figure 3
 
-    M_T, Σ_T = core.compute_mean_std(T_A, m)
-    μ_Q, σ_Q = core.compute_mean_std(T_B, m)
-
-    T_A[np.isnan(T_A)] = 0
-
     out = np.empty((l, 4), dtype=object)
 
     # Handle first subsequence, add exclusionary zone
-    if ignore_trivial:
-        P, I = stamp.mass(T_B[:m], T_A, M_T, Σ_T, 0, excl_zone)
-        PR, IR = stamp.mass(T_B[:m], T_A, M_T, Σ_T, 0, excl_zone, right=True)
+    if not np.isinf(μ_Q[0]):
```

Comment:
Can you please swap the order of the `if/else` statement so that it is?

```
if np.isinf(μ_Q[0]):
    .
else:
    .
```

This avoids having to perform some "mental gymnastics" to think about what `not np.isinf(μ_Q[0])` means